### ISSUE TITLE
[FR] Video Quality/Subtitles In App Settings

### ISSUE THREAD

**Issue by DarkCrypt** [BeamlakAschalew/cinemax#39]
### Steps to reproduce

No bug, feature request. (No feature request template for submission)

The video player already has options for choice of quality and subtitles but user must select their preferred choice everytime a video is played.
Ex. User must turn off subtitles for every video play.

**Feature Request:**
If possible, add global video quality and subtitle options in app settings page.

### Expected behavior

Thank

### Actual behavior

You

### Android version

Android 13

### Other details

Cinemax continuously gets better all the time. You're an awesome dev and always responsive, keep up the great work friend!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by BeamlakAschalew** [comment]
Yeah, I started doing that, check models/setting_preferences.dart, I will try to release it after 3 weeks, not sure though.
You can see the list I made for future features here: https://trello.com/b/FE3Cvlzr/cinemax-future-features.
I have other things to do now though, the features may not be added frequently.

**Comment by DarkCrypt** [comment]
Ok, sounds good. No hurry, whenever you put updates out is whenever you put updates out. Users need to understand it takes time and hard work. I sincerely respect and appreciate you!

I apologize for bothering you about features and improvements you already have planned. I didn't know about your trello board, I'll be sure to check that before making any future requests. Maybe include that in a README update? Up to you. Feel free to close this out whenever you want.
